pyproject.toml

```diff
@@ -43,4 +43,4 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.ruff]
-line-length = 120
+line-length = 99
```

Comment:
Let's leave this to be 120 please? We use wider displays than 99 today

Reply:
Do you mean physical displays, like monitors, or what?

Anyway, long lines are in general harder to read, I would prefer to have them even shorter 🤷‍♂️ 

I suggested 99 as a compromise, since it is the same compromise we came to in qrant-client.
And I think it is better to have the same formatting across the company projects. E.g., it will allow to have a ruff/black/etc be configured system-wise, and I find it convenient.
(I think we will consider switching to ruff in qdrant-client for the sake of congruence is well)